src/test/e2e/helpers/test-helper.ts

```diff
@@ -45,15 +73,44 @@ function httpApis(
 
     return {
         createFeature: (
-            name: string,
+            feature: string | CreateFeatureSchema,
             project: string = DEFAULT_PROJECT,
             expectedResponseCode: number = 201,
         ) => {
+            let body = feature;
+            if (typeof feature === 'string') {
+                body = {
+                    name: feature,
+                };
+            }
             return request
                 .post(`${base}/api/admin/projects/${project}/features`)
-                .send({
-                    name,
-                })
+                .send(body)
+                .set('Content-Type', 'application/json')
+                .expect(expectedResponseCode);
+        },
+
+        getFeatures(
+            name?: string,
+            expectedResponseCode: number = 200,
+        ): supertest.Test {
+            const featuresUrl = `/api/admin/features${name ? `/${name}` : ''}`;
+            return request
+                .get(featuresUrl)
+                .set('Content-Type', 'application/json')
+                .expect(expectedResponseCode);
+        },
+
+        getProjectFeatures(
```

Comment:
These ones also make me think the same as https://github.com/Unleash/unleash/pull/3348/files#r1145853611 - Having plural methods `getFeatures` and `getProjectFeatures` that don't take in a name, and singular `getFeature` and `getProjectFeature` ones that do. Whatever we decide there should maybe apply here as well, either separate into different methods or leave them be and accept that this is good enough.

Reply:
Yes, I feel the same way, but because this is test code and a thin layer on top of a REST API, I think we can let it be